utils/acpica/Makefile

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright (C) 2017 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=acpica
+PKG_VERSION:=20170531
+PKG_RELEASE:=1
+
+PKG_BASENAME:=$(PKG_NAME)-unix
+PKG_SOURCE:=$(PKG_BASENAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://acpica.org/sites/$(PKG_NAME)/files/$(PKG_SOURCE)
+PKG_MD5SUM:=cf00eec367b996219f640756895f414f
+PKG_HASH:=50155778cd1633dfca3443b8f8fd1ccc30e70e55ddece4d3c4fceafbbf1ab0e8
+PKG_MAINTAINER:=Philip Prindeville <philipp@redfish-solutions.com>
+
+PKG_LICENSE:=GPL-2.0
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_BASENAME)-$(PKG_VERSION)
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/acpica
+  SECTION:=utils
+  CATEGORY:=Utilities
+  DEPENDS:=
+  TITLE:=ACPI utilities for UNIX
+  URL:=https://01.org/linux-acpi/utilities
+endef
+
+define Package/acpica/description
+	Open Source utilities for ACPI including the ACPICA Machine Language
+	(AML) interpreter, a simulator, test suites, and a compiler to
+	translate ACPI Source Language (ASL) into AML.
+endef
+
+define Build/Configure
+endef
+
+MAKE_VARS += HOST=_LINUX \
+	     NOFORTIFY=TRUE
+
+define Build/Compile
+	+$(MAKE_VARS) \
+	$(MAKE) -C $(PKG_BUILD_DIR)/generate/unix \
+		$(TARGET_CONFIGURE_OPTS) \
+		CC="$(TARGET_CC)" \
+		LD="$(TARGET_LD)"
+endef
+
+#define Package/acpica/install
+#	+$(MAKE_VARS) \
+#	$(MAKE) -C $(PKG_BUILD_DIR)/generate/unix install \
+#		$(TARGET_CONFIGURE_OPTS) \
+#		INSTALLFLAGS="-m 755 -s" \
+#		DESTDIR=$(1)
+#endef
```

Comment:
I was thinking if we ever want to install more programs... though most of them are useful only for host-side development... But I can remove this.